Ladybird/CMakeLists.txt

```diff
@@ -258,7 +258,7 @@ include(CTest)
 if (BUILD_TESTING)
     add_test(
         NAME LibWeb
-        COMMAND ${CMAKE_CURRENT_BINARY_DIR}/../bin/headless-browser --resources "${SERENITY_SOURCE_DIR}/Base/res" --run-tests ${SERENITY_SOURCE_DIR}/Tests/LibWeb
+        COMMAND ${CMAKE_CURRENT_BINARY_DIR}/../bin/headless-browser --resources "${SERENITY_SOURCE_DIR}/Base/res" --run-tests ${SERENITY_SOURCE_DIR}/Tests/LibWeb --dump-failed-ref-tests
```

Comment:
I don't know if this is also going to affect CI. The point of putting a flag is to prevent needlessly dumping files on the test runner where nobody can ever view them.

Reply:
In theory I think they could become some kind of "build artifacts" or whatever.
Also, we could base64 encode them and dump to the log :^)

Reply:
Done!